conf/application.conf

```diff
@@ -331,10 +330,6 @@ assets {
   url = "http://localhost:9032/assets/"
```

Comment:
Pls can you make this on relative as well

Reply:
for some reason , it looks like its expecting full url , all other services use the same format
`https://github.tools.tax.service.gov.uk/HMRC/app-config-qa/search?utf8=%E2%9C%93&q=assets.url&type=Code`